lib/driverProviders/driverProvider.js

```diff
@@ -34,6 +34,7 @@ DriverProvider.prototype.getExistingDrivers = function() {
 DriverProvider.prototype.getNewDriver = function() {
   var builder = new webdriver.Builder().
     usingServer(this.config_.seleniumAddress).
+    usingWebDriverProxy(this.config_.seleniumProxy).
```

Comment:
Can you add a note to https://github.com/angular/protractor/blob/master/docs/referenceConf.js explaining what this config option does?